components/myforms/HTMLDownload/Table.tsx

```diff
@@ -49,7 +49,7 @@ const formatColumnAnswers = (
         </dt>
         <dd className="py-4">
           {response.map((subItem, subIndex) => (
-            <p key={`${parentIndex}-${subIndex}`}>{subItem.toString() || "-"}</p>
+            <p key={`${parentIndex}-${subIndex}`}>{String(subItem) || "-"}</p>
```

Comment:
Slightly safer than calling .toString(). I still have some baggage from Java and toStrings.. :) I forget why I'm explicitly calling an object's toString(), there must have been an edge case. If/when I remember I'll add a comment about it